src/sbr/SR.java

```diff
@@ -247,7 +247,7 @@ private Vertex nextBridgeLinkedStartVertex() {
     }
 
     private boolean canBeStart(Vertex sw) {
-        return (!isVisited(sw) && sw.isIn(currentWindow) && hasSuitableLinks(sw));
+        return (!isVisited(sw) && sw.isIn(currentWindow));
```

Comment:
Take a look at the commit message of the first commit ("Fixes comment out assertions").